static/app/views/profiling/profileSummary/index.tsx

```diff
@@ -0,0 +1,198 @@
+import {Fragment, useCallback, useMemo} from 'react';
+import {browserHistory} from 'react-router';
+import styled from '@emotion/styled';
+import {Location} from 'history';
+
+import DatePageFilter from 'sentry/components/datePageFilter';
+import EnvironmentPageFilter from 'sentry/components/environmentPageFilter';
+import IdBadge from 'sentry/components/idBadge';
+import * as Layout from 'sentry/components/layouts/thirds';
+import NoProjectMessage from 'sentry/components/noProjectMessage';
+import PageFilterBar from 'sentry/components/organizations/pageFilterBar';
+import PageFiltersContainer from 'sentry/components/organizations/pageFilters/container';
+import {Breadcrumb} from 'sentry/components/profiling/breadcrumb';
+import SentryDocumentTitle from 'sentry/components/sentryDocumentTitle';
+import SmartSearchBar, {SmartSearchBarProps} from 'sentry/components/smartSearchBar';
+import {MAX_QUERY_LENGTH} from 'sentry/constants';
+import {t, tct} from 'sentry/locale';
+import space from 'sentry/styles/space';
+import {PageFilters, Project} from 'sentry/types';
+import {defined} from 'sentry/utils';
+import {useProfileFilters} from 'sentry/utils/profiling/hooks/useProfileFilters';
+import {decodeScalar} from 'sentry/utils/queryString';
+import {MutableSearch} from 'sentry/utils/tokenizeSearch';
+import useOrganization from 'sentry/utils/useOrganization';
+import useProjects from 'sentry/utils/useProjects';
+import withPageFilters from 'sentry/utils/withPageFilters';
+
+import {ProfileSummaryContent} from './content';
+
+interface ProfileSummaryPageProps {
+  location: Location;
+  params: {
+    projectId?: Project['slug'];
+  };
+  selection?: PageFilters;
+}
+
+function ProfileSummaryPage(props: ProfileSummaryPageProps) {
+  const organization = useOrganization();
+  const {projects} = useProjects({
+    slugs: props.params.projectId ? [props.params.projectId] : [],
```

Comment:
Nit: used `defined` below for the similar functionality, I dont really care which one you want to use, just leaving a note in case you have a preference.